virttest/qemu_vm.py

```diff
@@ -92,6 +94,23 @@ def clean_tmp_files():
                                     'avocado-vt-vm-create.lock')
 
 
+def monitor_qemu_process_term(vm, term_watcher, exit_status):
+    """
+    Callback function to detect QEMU process non-zero exit status and
+    push VMExitStatusError to background error bus.
+    """
+    if exit_status != 0:
+        msg = "vm %s exited with non-zero status %s" % (vm, exit_status)
+        if term_watcher == "log":
```

Comment:
We already have the log line in debug.log, so do not have to do this one more time.